lib/src/view/analysis/tree_view.dart

```diff
@@ -162,124 +131,580 @@ class _InlineTreeViewState extends ConsumerState<AnalysisTreeView> {
           ),
         SliverFillRemaining(
           hasScrollBody: false,
-          child: Padding(
-            padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 10),
-            child: Wrap(
-              spacing: kInlineMoveSpacing,
-              children: moveWidgets,
+          child: _PgnTreeView(
+            root: root,
+            rootComments: rootComments,
+            params: (
+              shouldShowAnnotations: shouldShowAnnotations,
+              shouldShowComments: shouldShowComments,
+              currentMoveKey: currentMoveKey,
+              currentPath: currentPath,
+              notifier: () => ref.read(ctrlProvider.notifier),
             ),
           ),
         ),
       ],
     );
   }
+}
+
+// A group of parameters that are passed through various parts of the tree view
```

Comment:
Even for private members we should use the triple slash comments (with a first short line and empty line and description below). It improves code readability a lot, for instance in vim I press "K" and I see the code doc.

Reply:
Ah, a fellow vim user I see! 

Yeah that was an oversight, will fix